### ISSUE TITLE
can't install tilix on centos7.3

### ISSUE THREAD

**Issue by statquant** [gnunn1/tilix#1404]
Hello when I try to install tilix on centos7.3 I get the following issue:
```
bash) tilix
** (tilix:103187): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-1FscfPxAao: Connection refused
object.Error@(0): The function you are calling is not pressent in your version of GTK+.
----------------
??:? [0x8bc5b06e]
??:? [0x8bc647ba]
??:? [0x8bc3d542]
??:? [0x8ba5c67b]
??:? [0x8b7d4b41]
??:? [0x8b6612b7]
??:? [0x8b662bc4]
??:? [0x8b669e0c]
??:? [0x8b68bd6a]
??:? [0x8b68baf0]
??:? [0x8b696abc]
??:? [0x8b718a8c]
??:? [0x8b71ef28]
??:? [0x8b72001c]
??:? [0x8b726d4a]
??:? [0x8b728c62]
??:? [0x8b8d5ad0]
??:? g_closure_invoke [0x88e53907]
??:? [0x88e65a1c]
??:? g_signal_emit_valist [0x88e6dab0]
??:? g_signal_emit [0x88e6dd9e]
??:? [0x8b8d4c18]
??:? [0x8b7284b0]
??:? [0x8b8d4a32]
??:? ffi_call_unix64 [0x8890adcb]
??:? ffi_call [0x8890a6f4]
??:? g_cclosure_marshal_generic [0x88e54147]
??:? g_closure_invoke [0x88e53907]
??:? [0x88e65a1c]
??:? g_signal_emit_valist [0x88e6d79b]
??:? g_signal_emit [0x88e6dd9e]
??:? [0x8913c9e2]
??:? [0x8913e9a5]
??:? g_application_run [0x8913eb4b]
??:? [0x8b8d540a]
??:? [0x8b733c08]
??:? [0x8bc3d03e]
??:? [0x8bc3cf7d]
??:? [0x8bc3ce74]
??:? [0x8b7c6fe4]
??:? __libc_start_main [0x89f34b34]
??:? [0x8b62a319]
```

that's with 
```
bash) tilix --version
** (tilix:103992): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-1FscfPxAao: Connection refused
Versions
	Tilix version: 1.7.5
	VTE version: 0.42
	GTK Version: 3.14.13

Tilix Special Features
	Notifications enabled=0
	Triggers enabled=0
	Badges enabled=0
```

Is there a list of packages that assures tilix will work ?
Sorry to ask this but I am not an admin and I have to provide a list of packages for a chance to get it working.
I can run `yum list` though if it helps

**Comment by gnunn1** [comment]
Honestly it's probably time for me to look at dropping support for GTK 3.14 as that is getting positively ancient now.

**Comment by statquant** [comment]
I am perfectly fine to install anything, would you be kind enough to provide a list ?

**Comment by gnunn1** [comment]
Upgrade to centos 7.5 and it should be fine since it will have a newer GTK version or use Fedora which is more desktop oriented.

**Comment by statquant** [comment]
Argh, update the OS I cannot ask for :( I meant packages, sorry I wasn't clear.

**Comment by gnunn1** [comment]
I have fixed this, however the next release will be the last tilix release to support GTK 3.14 so your going to be out of luck after this, sorry.

**Comment by NeMO84** [comment]
@gnunn1 , thank you for fixing this!

Any general ETA for the next release being available?

**Comment by statquant** [comment]
@gnunn1  Thank you bery much that's great. Do you have some continuous integration process that's going to build the rpm or release this version, happy to follow the instructions from https://gnunn1.github.io/tilix-web/ for centos7

**Comment by gnunn1** [comment]
I don't have a fixed timeline in place but probably in the next week with or so.

**Comment by statquant** [comment]
Great would you kindly let us know ? Many thanks again
